buildSrc/src/main/java/com/uber/okbuck/composer/jvm/JvmIntegrationTestRuleComposer.java

```diff
@@ -0,0 +1,67 @@
+package com.uber.okbuck.composer.jvm;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableSet;
+import com.uber.okbuck.core.model.base.RuleType;
+import com.uber.okbuck.core.model.base.SourceSetType;
+import com.uber.okbuck.core.model.jvm.JvmTarget;
+import com.uber.okbuck.template.core.Rule;
+import com.uber.okbuck.template.jvm.JvmRule;
+
+import java.util.List;
+import java.util.Set;
+
+public final class JvmIntegrationTestRuleComposer extends JvmBuckRuleComposer {
```

Comment:
Missing JavaDocs?

Reply:
None of the composers got JavaDocs, I personally think that adding a java doc that would say `Class responsible for composing Jvm IntegrationTest Rules` doesn't add any information that the name itself already says

Reply:
For industry convention I personally think that the name should be `Factory` then, since the class is responsible for initialising objects. When first reading `Compose` one might assume that it does more than a `Factory`  since it was not named that way. But again it would fall off the scope of this PR

Reply:
Agreed